bears/php/PHPStanBear.py

```diff
@@ -0,0 +1,41 @@
+import os
+
+from coalib.bearlib.abstractions.Linter import linter
+
+
+@linter(executable='phpstan',
+        output_format='regex',
+        output_regex=r'(?P<line>\d+)  (?P<message>.*)')
+class PHPStanBear:
+    """
+    Checks the code with ``phpstan analyze``.
+    This can run it on multiple files and folders.
+    See <https://github.com/phpstan/phpstan> for more information.
+    """
+    LANGUAGES = {'PHP'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Syntax', 'Unused Code', 'Variable Misuse',
+                  'Undefined Element', 'Missing Import', 'Spelling'}
+
+    @staticmethod
+    def create_arguments(filename, file,
+                         config_file,
+                         phpstan_level: str='0',
+                         phpstan_config: str=''):
+        """
+        :param phpstan_config:
+            path to a custom configuration file.
+            When using a custom project config file,
+            you have to pass the phpstan_level argument
+            (default value 0 does not apply here).
+        :param phpstan_level:
```

Comment:
>When using a custom project config file, you have to pass the phpstan_level argument (default value 0 does not apply here).

If that's the case, please add check for this situation so that it doesn't result in a runtime error.